docs/doc_index.py

```diff
@@ -60,9 +60,10 @@ def name_to_sphinx(self):
 
 
 class OpReference:
-    def __init__(self, operator, docstring):
+    def __init__(self, operator, docstring, order = None):
         self.operator = operator
         self.docstring = docstring
+        self.order = 1000000 if order is None else order
```

Comment:
Unfortunately, Python has infinite integers and sorting by key (instead of compare function).